src/Driver/driver_configs.jl

```diff
@@ -130,6 +130,19 @@ struct DriverConfiguration{FT}
     end
 end
 
+function print_model_info(model)
+    msg = "AtmosModel composition\n"
+    for key in fieldnames(typeof(model))
+        msg =
+            msg * @sprintf(
+                "    %s = %s\n",
+                string(key),
+                string((getproperty(model, key)))
+            )
+    end
+    @info msg
```

Comment:
Nice, thanks for putting this back in ; One `Info` tag for the entire composition makes sense